src/features/filter-transactions/model.ts

```diff
@@ -14,19 +32,37 @@ export const sortOptions: TabItem<TabAssetsSort>[] = [
 ];
 
 export const blockOptions: TabItem<TabBlocksScreen>[] = [
-  { i18nKey: 'transactions.transactions', value: 'transactions' },
-  { i18nKey: 'transactions.transfers', value: 'transfers' },
-  { i18nKey: 'transactions.mints', value: 'mints' },
-  { i18nKey: 'transactions.burns', value: 'burns' },
-  { i18nKey: 'transactions.grants', value: 'grants' },
-  { i18nKey: 'transactions.revokes', value: 'revokes' },
+  { i18nKey: 'transactions.transactions', value: 'Transactions' },
+  { i18nKey: 'transactions.transfers', value: 'Transfer' },
+  { i18nKey: 'transactions.mints', value: 'Mint' },
+  { i18nKey: 'transactions.burns', value: 'Burn' },
+  { i18nKey: 'transactions.grants', value: 'Grant' },
+  { i18nKey: 'transactions.revokes', value: 'Revoke' },
 ];
 
 export const defaultOptions: TabItem<TabDefaultScreen>[] = [
-  { i18nKey: 'transactions.all', value: 'all' },
-  { i18nKey: 'transactions.transfers', value: 'transfers' },
-  { i18nKey: 'transactions.mints', value: 'mints' },
-  { i18nKey: 'transactions.burns', value: 'burns' },
-  { i18nKey: 'transactions.grants', value: 'grants' },
-  { i18nKey: 'transactions.revokes', value: 'revokes' },
+  { i18nKey: 'transactions.all', value: 'All' },
+  { i18nKey: 'transactions.transfers', value: 'Transfer' },
+  { i18nKey: 'transactions.mints', value: 'Mint' },
+  { i18nKey: 'transactions.burns', value: 'Burn' },
+  { i18nKey: 'transactions.grants', value: 'Grant' },
+  { i18nKey: 'transactions.revokes', value: 'Revoke' },
+];
+
+export const instructionOptions: TabItem<TabInstructions>[] = [
```

Comment:
Preferably to use screaming case for consts.

```suggestion
export const INSTRUCTION_OPTIONS: TabItem<TabInstructions>[] = [
```